### ISSUE TITLE
news tab: lists in blog posts have     list-style: none;

### ISSUE THREAD

**Issue by mfb** [EFForg/actioncenter-mobile#30]

**Comment by jeremytribby** [comment]
@mfb I don't think that news item is still in the feed, do you know of another with this behavior?

**Comment by mfb** [comment]
Yes! click on "Kafkaesque"

**Comment by mfb** [comment]
There is a numbered list at the bottom of this post.

**Comment by mfb** [comment]
(admittedly I created this numbered list myself because most deeplinks bloggers don't care about semantic markup ;)
